README.md

```diff
@@ -659,6 +659,8 @@ validate threshold       Validate the compliance and status counts of an HDF fil
     -T, --templateInline=     Flattened JSON containing your validation thresholds
                               (Intended for backwards compatibility with InSpec Tools)
     -i, --input               Input HDF JSON file
+    -m, --totalMinimum        Treat passed.total/failed.total/etc as an minimum requirement, if this
```

Comment:
Sorry not sure I understand the question, whether or not this should be true or false depends on the use case. 

Currently if you have the following:
```
passed:
  total: 19
```
it expects the number of passed controls to be exactly 19, instead of at least 19.